test/support/builders/metadata/base.js

```diff
@@ -61,6 +61,12 @@ module.exports = class BaseMetadataBuilder {
     return this
   }
 
+  moveTo (id /*: string */) /*: this */ {
```

Comment:
Wouldn't a `path` be more intuitive for the caller?

I would think so since even the author of this new builder method mistakenly used `.moveTo(qux.path)` in [`test/unit/merge.js:915`](https://github.com/cozy-labs/cozy-desktop/pull/1437/files#diff-d5f621b4689d0b6f0b95586c98caee11R915) :wink: 

Also, since `BaseMetadataBuilder#moveFrom()` takes a full doc, not a path nor an id, won't this new `#moveTo` method be a bit confusing? Although I'm not sure what would be the best solution... `moveToPath()`?